packages/paste-core/components/display-pill-group/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,126 @@
+import {Basic, CustomDisplayPillGroup} from '../stories/index.stories';
+
+expect.extend(matchers);
+
+describe('DisplayPillGroup', () => {
+  // Verifies that the correct aria attributes and semantics are met
+  describe('Rendered shape', () => {
+    it('should render correctly', () => {
+      const {getByTestId, getByText} = render(<Basic />);
+      expect(getByText('Tennis')).toBeDefined();
+
+      const group = getByTestId('display-pill-group');
+      expect(group.getAttribute('aria-label')).toBe('Your favorite sports:');
+      expect(group.tagName).toBe('UL');
+
+      const pillLink = getByTestId('display-pill-anchor');
+      expect(pillLink.getAttribute('href')).toBe('https://google.com');
+      expect(pillLink.getAttribute('rel')).toBe('noreferrer noopener');
+      expect(pillLink.getAttribute('target')).toBe('_blank');
+      expect(pillLink.tagName).toBe('A');
+
+      const pillBlank = getByTestId('display-pill-blank');
+      expect(pillBlank.tagName).toBe('DIV');
+    });
+  });
+
+  // Verifies the component is fully customizable
+  describe('Customization', () => {
+    it('should set an element data attribute for DisplayPillGroup & DisplayPill', () => {
+      const {getByTestId} = render(<Basic />);
+      const group = getByTestId('display-pill-group');
+      expect(group.getAttribute('data-paste-element')).toEqual('DISPLAY_PILL_GROUP');
+
+      const pillLink = getByTestId('display-pill-anchor');
+      expect(pillLink.getAttribute('data-paste-element')).toEqual('DISPLAY_PILL');
+
+      const pillBlank = getByTestId('display-pill-blank');
+      expect(pillBlank.getAttribute('data-paste-element')).toEqual('DISPLAY_PILL');
+    });
+
+    it('should set a custom element data attribute for DisplayPillGroup & DisplayPill', () => {
+      const {getByTestId} = render(
+        <DisplayPillGroup element="CUSTOM_GROUP" data-testid="group" aria-label="Your favorite sports:">
+          <DisplayPill element="CUSTOM_PILL" data-testid="pill">
+            A
+          </DisplayPill>
+        </DisplayPillGroup>
+      );
+      const group = getByTestId('group');
+      expect(group.getAttribute('data-paste-element')).toEqual('CUSTOM_GROUP');
+      const pill = getByTestId('pill');
+      expect(pill.getAttribute('data-paste-element')).toEqual('CUSTOM_PILL');
+    });
+
+    it('should add custom styles to DisplayPillGroup & DisplayPill', () => {
+      const {getByTestId} = render(<CustomDisplayPillGroup />);
+
+      const group = getByTestId('display-pill-group');
+      expect(group).toHaveStyleRule('margin', '0.75rem');
+
+      const pillLink = getByTestId('display-pill-anchor');
+      expect(pillLink).toHaveStyleRule('background-color', 'rgb(231,220,250)');
+
+      const pillBlank = getByTestId('display-pill-blank');
+      expect(pillBlank).toHaveStyleRule('background-color', 'rgb(231,220,250)');
+    });
+    it('should add custom styles to custom element Avatar', () => {
```

Comment:
```suggestion
    it('should add custom styles to custom element Display Pill Group', () => {
```

Reply:
What is the difference between this set of tests and the ones above for custom styles?

Reply:
One uses the default `element` prop and the other passes in a custom `element` prop for an instanced restyle.